### PR TITLE
OCPBUGS-35031: restore missing CloseButton styles

### PR DESCRIPTION
**Pull request by rhamilto**
Restores styles erroneously deleted in https://github.com/openshift/console/pull/13588

### CODE REVIEW

File: frontend/packages/console-shared/src/components/close-button/CloseButton.scss [hunk 1]
```diff
@@ -1,12 +1,5 @@
-.co-close-button {
-
-  &--float-right {
```
Comment (rhamilto) [inline frontend/packages/console-shared/src/components/close-button/CloseButton.scss:3]: `--float-right` is no longer needed as PatternFly's `<CloseButton>` has `float: inline-end;` set on it.

**Review by sg00dwin — APPROVED** [review]
/lgtm

**Pull request merged**